notes.md

```diff
@@ -0,0 +1,48 @@
+# Special rules this test suite implements
```

Comment:
thanks for leaving notes here, this would probably be better as a standard README https://github.com/RichardLitt/standard-readme